tests/unit_tests/blas/include/reference_blas_templates.hpp

```diff
@@ -224,7 +224,7 @@ template <typename fp>
 static void gemm(CBLAS_LAYOUT layout, CBLAS_TRANSPOSE transa, CBLAS_TRANSPOSE transb, const int *m,
                  const int *n, const int *k, const fp *alpha, const fp *a, const int *lda,
                  const fp *b, const int *ldb, const fp *beta, fp *c, const int *ldc);
-
+#ifdef NOT_HIPSYCL
```

Comment:
could you use the `ENABLE_HALF_ROUTINES` here and get rid of the explicit `NOT_HIPSYCL` macro?
or instead just use a macro that is defined by hipSYCL anyways?

Reply:
Ohh.. this shouldn't be here at all... this should be in #6 ... Thanks for noticing :)